chromium_src/ui/native_theme/native_theme_android.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_UI_NATIVE_THEME_NATIVE_THEME_ANDROID_H_
+#define BRAVE_CHROMIUM_SRC_UI_NATIVE_THEME_NATIVE_THEME_ANDROID_H_
+
+#define GetInstanceForNativeUi \
+  GetInstanceForNativeUi();    \
+  static ui::NativeTheme* GetInstanceForNativeUi_ChromiumImpl
```

Comment:
Nit: since we aren't calling the upstream implementation this could be renamed to `GetInstanceForNativeUi_UnUsed`, but totally up to you.